packages/builder/src/pages/builder/app/[application]/_layout.svelte

```diff
@@ -83,12 +95,43 @@
   <div class="root">
     <div class="top-nav">
       <div class="topleftnav">
-        <Icon
-          size="M"
-          name="ArrowLeft"
-          hoverable
-          on:click={() => $goto("../../portal/apps")}
-        />
+        <ActionMenu>
+          <div slot="control">
+            <Icon size="M" hoverable name="ShowMenu" />
+          </div>
+          <MenuItem on:click={() => $goto("../../portal/apps")}>
+            Exit to portal
+          </MenuItem>
+          <MenuItem
+            on:click={() => $goto(`../../portal/overview/${application}`)}
+          >
+            Overview
+          </MenuItem>
+          <MenuItem
+            on:click={() =>
+              $goto(`../../portal/overview/${application}?tab=Access`)}
+          >
+            Access
+          </MenuItem>
+          {#if isPublished}
+            <MenuItem
```

Comment:
Yep I completely agree - we nearly always navigate by handling click actions which is just a worse experience for the user, since you can't do things like opening in a new tab, or copying the URL. That's a bigger problem in general which I'd like to address at some stage.

There's no reason we can't improve this in future as routify has a `$url` helper (I think that's what it's called) which will spit out an absolute URL from that same relative URL input.

Not one to address for now, but I agree.